jekyll/_posts/2019-07-17-version-0202-released.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "Version 0.20.2 released"
+author: The Nim Team
+---
+
+The Nim team is happy to announce version 0.20.2, which is our second
+release candidate for version 1.0.
+
+To read more about version 0.20 and our plans for version 1.0 and beyond,
+check the [previous release notes](https://nim-lang.org/blog/2019/06/06/version-0200-released.html).
+This is mostly a bugfix release of version 0.20.0 (1.0 RC1).
+
+The only new feature is that `toOpenArray`, our slicing operator,
+is now available for JavaScript backend.
+
+Nim now bootstraps with `--styleCheck:error` flag, meaning that the compiler
+and the standard library are now style checked and they won't compile if
+different styles (e.g. `fooBar` and `foo_bar`) are used.
+If you would like to style check your own nimble package, but not its
+dependencies, you can use `--styleCheck:hint`, making your package consistent
```

Comment:
So I'll get hundreds of hints from the packages I depend on as well?

Reply:
No, you only get hints for the "main" nimble package/program that you compile.